bond/bond_test.go

```diff
@@ -17,9 +17,11 @@ package main
 import (
 	"fmt"
 	"github.com/containernetworking/cni/pkg/skel"
+	"github.com/containernetworking/cni/pkg/types"
 	types020 "github.com/containernetworking/cni/pkg/types/020"
 	types040 "github.com/containernetworking/cni/pkg/types/040"
 	types100 "github.com/containernetworking/cni/pkg/types/100"
+	"strconv"
```

Comment:
move this together with fmt and add an space to separate them from the github ones

Reply:
moved it